tedana/workflows/tedana.py

```diff
@@ -248,6 +228,10 @@ def tedana_workflow(data, tes, mask=None, mixm=None, ctab=None, manacc=None,
     tes = [float(te) for te in tes]
     n_echos = len(tes)
 
+    # Coerce gscontrol to list
+    if not isinstance(gscontrol, list):
```

Comment:
If any values were supplied here it will be a list since you used `nargs='+'`. Is this just to check that it's not `None` ?

Reply:
That's right, since `None` should be the default. Plus I'm using `if x in gscontrol` which would break if `gscontrol` was left as `None`.

Reply:
Could we make this explicit by having None return an empty list, rather than a list with a value of `None` ? Like:
```
if gscontrol is None:
    gscontrol = ['']
```

Reply:
That will make things easier for the CLI, but not for the Python function, since users could still input strings like `tedana_workflow(..., gscontrol='gsr')`.